QUEST_LV_0100.tsv

```diff
@@ -119,82 +119,82 @@ QUEST_LV_0100_20150317_000118	${memo X}Isn't it better to use the cable car rath
-QUEST_LV_0100_20150317_000127	Please unleash the seals with this Key of the Great King at the Awaiting Stone Bridge.{nl}You will know what to do next.
-QUEST_LV_0100_20150317_000128	I found Irene, but I don't know how to rescue her.{nl}She is tied at the 2nd Architects' Site in Rodoma, but I can't rescue her myself...
-QUEST_LV_0100_20150317_000129	Assistant Irene
-QUEST_LV_0100_20150317_000130	{memo X}I can't move because of the barrier.{nl}Please break the three barriers at the Rodoma Architects' Site.
-QUEST_LV_0100_20150317_000131	Not done yet? Damn...
-QUEST_LV_0100_20150317_000132	I didn't expect that I would fall into this kind of thing.{nl}Damn, I should not have followed him.
-QUEST_LV_0100_20150317_000133	Mercenary Glen
-QUEST_LV_0100_20150317_000134	So many researchers are disappearing these days.{nl}I'm trying to find some clues by defeating Saugas. Can you help us?
-QUEST_LV_0100_20150317_000135	Saugas distribute rewards equally among themselves.{nl}Maybe we can find some clues there.
-QUEST_LV_0100_20150317_000136	Fortunately they are very old.{nl}That means it's not due to monsters.
-QUEST_LV_0100_20150317_000137	This is a very dangerous place and many people have gone missing here.{nl}But that is not the main problem here.{nl}
-QUEST_LV_0100_20150317_000138	Sometimes when certain people disappear no one else notices.{nl}It's as if all memories of those who disappear are erased. Isn't it strange?{nl}
+QUEST_LV_0100_20150317_000122	$That barrier...hmm. When did it form?{nl}It has been there for quite some time now, but I can't remember exactly how long.{nl}
+QUEST_LV_0100_20150317_000123	$I'm sure that barrier is responsible for the increasing number of monsters lately.{nl}
+QUEST_LV_0100_20150317_000124	$We don't have many soldiers guarding this area.{nl}Who in their right mind would come here in this day and age?{nl}
+QUEST_LV_0100_20150317_000125	$Rolandas Jonas
+QUEST_LV_0100_20150317_000126	$No one knows what the Great King Zachariel tried to protect.{nl}And I'm sure that we're not the only ones who want to solve that mystery.
+QUEST_LV_0100_20150317_000127	$Hurry, use the Great King's Key to release the seals.{nl}You should know what to do next afterwards.
+QUEST_LV_0100_20150317_000128	$I found Irene, but I don't know how to rescue her.{nl}She is imprisoned in Rodoma 2nd Ruins, but I can't rescue her myself...
+QUEST_LV_0100_20150317_000129	$Assistant Irene
+QUEST_LV_0100_20150317_000130	{memo X}I can't move because of the barrier.{nl}Please break the three barriers in Rodoma Ruins.
+QUEST_LV_0100_20150317_000131	$Are you not done yet? Please hurry...
+QUEST_LV_0100_20150317_000132	$I didn't expect that I would be taken into this kind of thing.{nl}Oh, the mess I caused from chasing that boy.
+QUEST_LV_0100_20150317_000133	$Mercenary Glen
+QUEST_LV_0100_20150317_000134	$So many researchers are disappearing these days.{nl}I'm trying to find some clues by defeating some Saugas. Can you help us?
```

Comment:
Sauga is a boss monster. Omit the 'some' and change 'Saugas' to 'Sauga'. Maybe change 'clues' to 'a clue'?